BootloaderCorePkg/Tools/GenContainer.py

```diff
@@ -703,7 +723,7 @@ def main():
     cmd_display.add_argument('-t', dest='img_type',  type=str, default='CLASSIC', help='Container Image Type : [NORMAL, CLASSIC, MULTIBOOT]')
     cmd_display.add_argument('-o', dest='out_path',  type=str, default='.', help='Container output directory/file')
     cmd_display.add_argument('-k', dest='key_path',  type=str, default='', help='Input key directory/file')
-    cmd_display.add_argument('-ht', dest='hash_type', type=str, choices=['SHA2_256', 'SHA2_384'], default='SHA2_256', help='Hash Alg for signing')
+    cmd_display.add_argument('-ht', dest='hash_type', type=str, choices=['SHA2_256', 'SHA2_384'], default='SHA2_256', help='Hash Algorithm Type')

```

Comment:
let us use lower cases  "Hash algorithm type" for all occurances

Reply:
Maurice, have question. Currently we support auth types param - 
(SHA2_256, SHA2_384, RSA2048, RSA3072).  Internally these are mapped to (SHA2_256, SHA2_384, RSA2048SHA256, RSA3072SHA384)

Instead of using two params **auth** and **hash_type**, could we use only **auth**  and recommend user to use auth (SHA2_256, SHA2_384, **RSA2048SHA256**, **RSA3072SHA384**) at container create() and Sign()? 
This change would be need an update in all boardconfig files and stitch configs where only "RSA2048" is used.

Reply:
Yes, that is fine with me.